packages/babel-parser/src/parser/expression.js

```diff
@@ -925,6 +925,14 @@ export default class ExpressionParser extends LValParser {
       const node = this.startNode();
       this.next();
```

Comment:
I _think_ you can save `this.state.pos` (or something like that) just before this and just after this and compare them, rather than working with the AST nodes.

Reply:
Actually, save `this.state.end` and then check `this.state.start` after calling `next()`.

Reply:
@macabeus Sorry for not giving any information. Logan's suggestion is good.

Reply:
@nicolo-ribaudo No problem 😋